packages/core/src/components/chip/chip.tsx

```diff
@@ -86,24 +86,34 @@ export class Chip {
   private getCloseButton() {
     return (
       <div class="close-button-container">
-        <button
-          type="button"
-          class="btn btn-invisible-secondary btn-icon btn-oval close-button"
-          onClick={(event) => {
-            this.close.emit(event);
-            this.closeChip.emit(event);
-            event.stopPropagation();
-          }}
-        >
-          {this.variant === 'custom' ? (
-            <i
-              class="glyph glyph-16 glyph-close-small"
-              style={{ color: this.color }}
-            />
-          ) : (
-            <ix-icon name={'close-small'} size={'16'} />
-          )}
-        </button>
+        {this.variant === 'custom' ? (
+          <ix-icon-button
+            type="button"
+            variant="Secondary"
+            icon="close-small"
+            size="16"
+            style={{ color: this.color }}
+            ghost
+            onClick={(event) => {
+              this.close.emit(event);
+              this.closeChip.emit(event);
+              event.stopPropagation();
+            }}
+          ></ix-icon-button>
+        ) : (
+          <ix-icon-button
+            type="button"
+            variant="Secondary"
+            icon="close-small"
+            size="16"
+            ghost
+            onClick={(event) => {
+              this.close.emit(event);
+              this.closeChip.emit(event);
+              event.stopPropagation();
+            }}
+          ></ix-icon-button>
+        )}
```

Comment:
I don't think the doublicated code is required here

Possible solution (not tested)
```
<ix-icon-button style={ this.variant === 'custom' ? { color: this.color } : {}}
```